### DISCUSSION TITLE
Previous page reload

### DISCUSSION THREAD

**Discussion by suoc** [community/community#13286] [Other Feature Feedback, Questions, & Ideas]
Enter a link, go back to the previous page, the old page will be reloaded, and the position will be lost.

**Comment by jfullstackdev** [comment]
yes, there is a similar request, much like the feed of Twitter to return where you left off, by now, you can simply right click it in order to retain the feed page

but 

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
